tasks/main.yml

```diff
@@ -5,14 +5,14 @@
 - name: test to see if selinux is running
   command: getenforce
   register: sestatus
-  changed_when: false
-  ignore_errors: true
+  failed_when: false
+  always_run: true
```

Comment:
wouldn't you like better to add into all dependent tasks: `when: sestatus is defined` ? It'd give same result and you don't need always_run anymore